src/cli.ts

```diff
@@ -12,6 +12,76 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {server} from './server';
+import * as http from 'http';
+import * as minimist from 'minimist';
+import * as prpl from './prpl';
 
-server().listen(8080);
+const defaults = {
+  host: '127.0.0.1',
+  port: '8080',
+  root: '.',
+};
+
+const opts = {
+  string: ['host', 'port', 'root'],
+  boolean: ['help'],
+  default: defaults,
+  unknown: (arg: string) => {
+    throw new Error(`unknown arg: "${arg}"`);
+  },
+};
+
+const color = {
+  reset: '\x1b[0m',
+  magenta: '\x1b[35m',
+  blue: '\x1b[34m',
+};
+
+const help = `
+${color.magenta}prpl-server${color.reset}
+
+https://github.com/Polymer/prpl-server-node
+
+--host	Listen on this hostname (default ${defaults.host}).
+--port	Listen on this port; 0 for random (default ${defaults.port}).
+--root	Serve files relative to this directory (default ${defaults.root}).
+--help	Print this help text.`;
```

Comment:
We've been standardizing on `command-line-args` as much as possible, just so that we don't have N different arg parsers to deal with.

It will handle documenting args like this, but it gives you the option to display a header and customize it in various ways.